### PR TITLE
UI backend

### PR DESCRIPTION
**Pull request by TheLimeFreak**
Added the user interface and input handler.

### CODE REVIEW

**Review by KimAndreG — APPROVED** [review]
look good to me.

**Comment by BelminH** [comment]
Looks good

**Review by BelminH — COMMENTED** [review]
Looks good

**Review by OdinBjerga — COMMENTED** [review]
Ser bra ut 💯 🤣 👌

File: Deliverables/ObligatoriskOppgave1.md [hunk 1]
```diff
@@ -1 +1 @@
-# Gruppe 8
\ No newline at end of file
+# Gruppe 8 navn
```
Comment (mrtineide) [inline Deliverables/ObligatoriskOppgave1.md]: Denne burde ikke endres med denne PR-en

Comment (TheLimeFreak) [reply Deliverables/ObligatoriskOppgave1.md]: Ok, i will change it

**Pull request merged**